shared/fs/row/drop-target.desktop.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+import * as React from 'react'
+import {Box, Icon} from '../../common-adapters'
+import * as Styles from '../../styles'
+
+type State = {|
+  showDropOverlay: boolean,
+|}
+
+const DropOverlay = ({onDragLeave, onDrop}) => (
+  <Box style={styles.dropOverlayStyle} onDragLeave={onDragLeave} onDrop={onDrop}>
+    <Icon type="icon-dropping-file-48" />
+  </Box>
+)
+
+class DropTarget extends React.PureComponent<any, State> {
```

Comment:
Mind type the props rather than using `any`?

Reply:
Done.